src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/JavaNCSSCheck.java

```diff
@@ -75,6 +75,15 @@
  * <pre>
  * &lt;module name="JavaNCSS"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void test () {
+ *   System.out.println("Line 1");
+ *   ...
+ *   System.out.println("Line 50") // OK
+ *   System.out.println("Line 51") // Violation as the method crosses 50 lines of uncommented code
```

Comment:
1. `Violation` -> lower case
2. replace `as` with comma, comment should be separated by comma from "violation"

Reply:
Done